core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/Stain.java

```diff
@@ -0,0 +1,42 @@
+package uk.ac.bbsrc.tgac.miso.core.data;
+
+import java.io.Serializable;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "Stain")
+
+public class Stain implements Serializable {
+  private static final long serialVersionUID = 1L;
+  private String stainGroup;
```

Comment:
StainGroup seems to be singular both on the model and in the DB. Is this wrong? If a Stain belongs to only one StainGroup, then there are less StainGroups than Stains. If we're constraining Stains to a DB table, shouldn't StainGroups be constrained too? Especially if submenus may be necessary?